src/sidebar/services/frame-sync.js

```diff
@@ -109,9 +113,15 @@ export class FrameSyncService {
     watch(
       this._store.subscribe,
       [() => this._store.allAnnotations(), () => this._store.frames()],
+      /**
+       * @param {[Annotation[], Frame[]]} arg0
+       * @param {[Annotation[]]} arg1
```

Comment:
```suggestion
       * @param {[Annotation[], Frame[]]} current
       * @param {[Annotation[]]} prev
```

Even though `arg0` and `arg1` are not referenced here, we might as well give them more meaningful names for the reader's benefit.

Reply:
👍🏼